core/src/main/java/hudson/util/DOSToUnixPathHelper.java

```diff
@@ -0,0 +1,75 @@
+package hudson.util;
+
+import hudson.EnvVars;
+import hudson.Util;
+
+import java.io.File;
+
+import static hudson.Util.fixEmpty;
+
+class DOSToUnixPathHelper {
```

Comment:
Can't a plugin with a class in `hudson.util` access this, possibly even accidentally?

`@Restricted` would make it clearer IMO.

Reply:
@daniel-beck: is this what you're looking for https://github.com/jsoref/jenkins/pull/4? If so, I can merge that into this PR.

Reply:
Yes, that would take care of it.